### DISCUSSION TITLE
show misconfig ID in table view

### DISCUSSION THREAD

**Discussion by itaysk** [aquasecurity/trivy#7597] [Ideas]
### Description

the table format of misconfiguration scanner doesn't show the ID of each finding, which makes it hard to refer to the ID for e.g ignore it. For reference, the vulnerability scanner shows CVE ID in table view.

### Target

None

### Scanner

Misconfiguration

**Comment by fatihtokus** [comment]
Hi @itaysk ,

Is this yellow column what you need?

![image](https://github.com/user-attachments/assets/39236ce1-87f2-4d53-bf3d-2a67ae871e38)

**Reply by afdesk** [reply]
@fatihtokus  I think Itay meant `table` (default) output.
Now misconfig results don't contain IDs:

```sh
$ trivy fs --scanners misconfig .
...
Tests: 120 (SUCCESSES: 113, FAILURES: 7, EXCEPTIONS: 0)
Failures: 7 (UNKNOWN: 0, LOW: 5, MEDIUM: 2, HIGH: 0, CRITICAL: 0)

MEDIUM: Container 'main' of StatefulSet 'trivy' should set 'securityContext.allowPrivilegeEscalation' to false
════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════
A program inside the container can elevate its own privileges and run as root, which might give the program control over the container and node.

See https://avd.aquasec.com/misconfig/ksv001
──────────────────────────────────────────────
```

**Reply by itaysk** [reply]
yes :) although it's not a table, the flag is called table

**Reply by fatihtokus** [reply]
gotcha :)
